runtime/lua/vim/lsp.lua

```diff
@@ -1458,11 +1474,7 @@ local function adjust_start_col(lnum, line, items, encoding)
     end
   end
   if min_start_char then
-    if encoding == 'utf-8' then
-      return min_start_char
-    else
-      return vim.str_byteindex(line, min_start_char, encoding == 'utf-16')
-    end
+    return util.str_byteindex(line, min_start_char, encoding)
```

Comment:
How does this handle the utf-8 case?

Reply:
It does, but I don't like wrapping an api function with a function sharing the same name